### DISCUSSION TITLE
version忘记改了

### DISCUSSION THREAD

**Discussion by Archer-Tatsu** [hgjazhgj/FGO-ExpBall#5] [General]
https://github.com/hgjazhgj/FGO-ExpBall/blob/e1d4bb33c9985aab4e70e17207dd2d8f4a169fc8/FGO-ExpBall/fgoConst.py#L1

以为自己pull错代码了，发现这里好像没改……

**Comment by hgjazhgj** [comment]
多谢提醒  
~~其实这种事情已经发生很多次了,不知道有没有技术手段强制我master上的每个commit中的version都发生变化...~~
